traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/postgres.go

```diff
@@ -133,11 +133,32 @@ func (p *Postgres) DeleteDNSSECKeys(cdnName string, tx *sql.Tx, ctx context.Cont
 }
 
 func (p *Postgres) GetURLSigKeys(xmlID string, tx *sql.Tx, ctx context.Context) (tc.URLSigKeys, bool, error) {
-	return tc.URLSigKeys{}, false, notImplementedErr
+	tvTx, dbCtx, cancelFunc, err := p.beginTransaction(ctx)
+	if err != nil {
+		return tc.URLSigKeys{}, false, err
+	}
+	defer p.commitTransaction(tvTx, dbCtx, cancelFunc)
+	return getURLSigKeys(xmlID, tvTx)
```

Comment:
I think we may want to pass `ctx` into these function as well in order to use `checkErrWithContext` after attempting the DB query. That function is basically there to give us better context in the error message in case the client closed the connection or the query timed out -- both of which _cancel_ the context, cancelling the DB query.